.config/nextest.toml

```diff
@@ -2,11 +2,25 @@
 fail-fast = false
 
 [test-groups]
+mysql = { max-threads = 1 } # Explanation below
 setup = { max-threads = 1 }  # Serialize the setup steps
 containerized = { max-threads = 8 }  # Don't use too much memory
 engine = { max-threads = 2 }  # Engine tests are very memory-hungry
 database = { max-threads = 8 }  # Don't use too much memory
 
+# TODO: Delete this workaround when this PR is merged:
+#       - Fix: nextest cleanup race condition by bonega
+#         https://github.com/launchbadge/sqlx/pull/3334
+#
+# NOTE: There is an incompatibility between nextest and sqlx:
+#       - nextest implies multiprocessing,
+#       - while sqlx has a lock on cleanup within the current process
+#         (https://github.com/launchbadge/sqlx/pull/2640#issuecomment-1659455042).
+[[profile.default.overrides]]
+filter = "test(::mysql::)"
```

Comment:
This filter would cover both unit tests and e2e tests. We only have problem with e2e tests. Can we improve tagging/filter to narrow the selection?

Reply:
> We only have problem with e2e tests

Not really -- the two groups of tests affect each other. It has to do with the sqlx per-test-binary cleanup, which only shows up on E2E because they take longer to run.

> Can we improve tagging/filter to narrow the selection?

But I'll double-check that possibility

Reply:
I double-checked, this solution doesn't work for us -- since the problem is not with the e2e tests themselves in particular, but with sqlx-per-binary (test binary) cleanup